src/utils/style-formatter.js

```diff
@@ -29,8 +29,13 @@ const getColor = ({ useColorExpression, colorExpression, color }, defaultColor,
   return resolvedColor === 'none' ? defaultColor : resolvedColor;
 };
 
-const setFontsize = (textElement, newFontsize, fontFamily) => {
-  textElement.setAttribute('style', `white-space: nowrap; font-size: ${newFontsize}px; font-family: ${fontFamily};`);
+const setFontsize = (textElement, newFontsize, fontFamily, addFlex, alignment) => {
```

Comment:
Could only add `aligment` as parameter, then check if it exists instead of using `addFlex`

Reply:
That would work probably. However flex is added to have the icon and text vertically aligned in a nice way

Reply:
Is there any reason not to use flex when not using an icon?

Reply:
Not really I think, but I removed the alignment from the button so we would need it here now